### ISSUE TITLE
Fandango video playback in normal view and player control bar after switching to full screen 

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/firefox-tv#2810]
## Steps to reproduce
1. From the Pinned Tiles section navigate to Fandango
2. Tap on any of the movies 
3. Tap on the upper right Movie pic which has a play sign
4. After video playback starts try to navigate to the bottom of the web page
5. Tap on the video player area to activate the control bar and tap on the full screen button

### Expected behavior
For step 4 the user shouldn't be able to scroll further down
For step 5 after switching to full screen view the control bar should be properly displayed

### Actual behavior
At step 4 empty space left between the control bar and the bottom of the page
At step 5 quite oversized control buttons

### Device information
* Fire TV device: Fire TV Pendant 4k
* Latest affected Firefox version: v4.4LAT2
* Earliest affected Firefox version: v4.4LAT2
* Turbo Mode: ON

### Notes
►Video scroll and empty space  
![20190906_103135_1](https://user-images.githubusercontent.com/51314259/64412870-0db43b80-d099-11e9-839d-56f778a7c777.gif)

►Video oversized control buttons
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/51314259/64412844-faa16b80-d098-11e9-9135-92fecdd34ea4.gif)

**Comment by athomasmoz** [comment]
@AndiAJ Have you seen this on any other sites? Or is it specific to Fandango

**Comment by AndiAJ** [comment]
@athomasmoz , no,haven't seen this video control bar on other websites, it's specific to Fandango
